lib/eslintrc.json

```diff
@@ -8,18 +8,131 @@
     "standard",
     "standard-jsx",
     "plugin:flowtype/recommended",
-    "plugin:jsx-a11y/recommended"
+    "plugin:jest/recommended"
   ],
   "parser": "babel-eslint",
   "plugins": [
     "flowtype",
+    "jest",
     "jsx-a11y"
   ],
   "rules": {
+    "complexity": ["warn", 10],
+    "no-magic-numbers": ["warn", {
+      "ignore": [-1, 0, 1]
+    }],
     "prefer-const": ["warn", {
       "destructuring": "all",
       "ignoreReadBeforeAssign": false
-    }]
+    }],
+    "sort-imports": "warn",
+    "sort-keys": "warn",
```

Comment:
+1 for the sort plugins.  I've already been using the sort-keys.

Reply:
The `sort-imports` one I think is actually a bit weird and I've tried it, but stopped using it.  I've been sorting based off of package or filename which I don't think this plugin allows.

Also, I'd highly recommend adding `sort-vars` as well.

Reply:
It is weird. I like sorting by packages from external libraries first then internal files. Also, I prefer sorting by the package name instead of the var name. This is the default eslint rule...there may be a plugin for this. I'll remove and look for another one.

Also, will add sort-vars. That's simple and preferable.